apps/app/package.json

```diff
@@ -35,110 +34,96 @@
 	"devDependencies": {
-		"@types/deep-extend": "0.4.32",
-		"@types/koa": "^2.13.5",
-		"@types/koa-bodyparser": "^4.3.10",
-		"@types/koa__router": "^12.0.0",
-		"@types/lodash": "^4.14.189",
+		"@types/deep-extend": "0.6.2",
+		"@types/jest": "^29.5.13",
+		"@types/koa": "^2.15.0",
+		"@types/koa-bodyparser": "^4.3.12",
+		"@types/koa__router": "^12.0.4",
 		"@types/moment": "^2.13.0",
-		"@types/react": "^18.0.25",
-		"@types/react-dom": "^18.0.9",
+		"@types/react": "^18.3.11",
+		"@types/react-dom": "^18.3.0",
 		"@types/react-tabs": "^5.0.5",
-		"@types/react-toggle": "^4.0.3",
+		"@types/react-toggle": "patch:@types/react-toggle@npm%3A4.0.5#~/.yarn/patches/@types-react-toggle-npm-4.0.5-c40bf309fb.patch",
+		"@types/semver": "^7.5.8",
+		"@vitejs/plugin-react": "^4.3.2",
 		"@welldone-software/why-did-you-render": "^7.0.1",
-		"concurrently": "^7.5.0",
-		"css-loader": "^6.7.2",
-		"electron": "^19.1.6",
-		"electron-builder": "^23.6.0",
+		"concurrently": "^7.6.0",
+		"electron": "32.2.0",
+		"electron-builder": "24.13.3",
 		"electron-notarize": "^1.2.2",
-		"html-webpack-plugin": "^5.5.0",
-		"nodemon": "^2.0.20",
-		"sass": "^1.56.1",
-		"sass-loader": "^13.2.0",
-		"style-loader": "^3.3.1",
-		"ts-loader": "^9.4.1",
+		"jest": "^29.7.0",
+		"nodemon": "^2.0.22",
+		"sass": "^1.79.4",
+		"ts-jest": "^29.2.5",
 		"typed-emitter": "^2.1.0",
-		"webpack": "^5.75.0",
-		"webpack-cli": "^4.10.0",
-		"webpack-dev-server": "^4.11.1"
+		"vite": "^5.4.8"
 	},
-	"prettier": "@sofie-automation/code-standard-preset/.prettierrc.json",
 	"dependencies": {
-		"@babel/core": "^7.22.1",
-		"@emotion/react": "^11.10.5",
-		"@emotion/styled": "^11.10.5",
-		"@feathersjs/errors": "^5.0.8",
-		"@feathersjs/feathers": "^5.0.8",
-		"@feathersjs/koa": "^5.0.8",
-		"@feathersjs/socketio": "^5.0.8",
-		"@feathersjs/socketio-client": "^5.0.8",
+		"@babel/core": "^7.25.8",
+		"@emotion/react": "^11.13.3",
+		"@emotion/styled": "^11.13.0",
+		"@feathersjs/errors": "^5.0.30",
+		"@feathersjs/feathers": "^5.0.30",
+		"@feathersjs/koa": "^5.0.30",
+		"@feathersjs/socketio": "^5.0.30",
+		"@feathersjs/socketio-client": "^5.0.30",
 		"@fontsource/barlow": "^4.5.9",
 		"@fontsource/barlow-condensed": "^4.5.9",
 		"@fontsource/barlow-semi-condensed": "^4.5.10",
-		"@koa/router": "^12.0.0",
-		"@mui/icons-material": "^5.10.14",
-		"@mui/material": "^5.10.14",
+		"@koa/router": "^13.1.0",
+		"@mui/icons-material": "^5.16.7",
+		"@mui/material": "^5.16.7",
 		"@shared/api": "^0.12.0-alpha.5",
 		"@shared/lib": "^0.12.0-alpha.5",
 		"@shared/models": "^0.12.0-alpha.5",
 		"@shared/server-lib": "^0.12.0-alpha.5",
 		"@shared/tsr-bridge": "^0.12.0-alpha.5",
-		"@sofie-automation/sorensen": "^1.4.2",
-		"axios": "^1.1.3",
-		"bufferutil": "^4.0.7",
-		"casparcg-connection": "^6.0.6",
-		"classnames": "^2.3.2",
+		"@sofie-automation/sorensen": "patch:@sofie-automation/sorensen@npm%3A1.4.3#~/.yarn/patches/@sofie-automation-sorensen-npm-1.4.3-a11a53b994.patch",
```

Comment:
What's up with the patches? Just esm-module fixes?
Should we PR the changes to the upstream modules as well?

Reply:
The sorensen one should be upstreamed, and is an esm fix.
The formik one is an incompatibility with updated react types, probably will go away when updating formik, but I didn't want to do a major bump of that yet.
The react-toggle ones are esm fixes, but upstream hasnt had a commit in 2 years, so I suspect trying to upstream it won't get anywhere.